doc/cascadia/Keybindings-Arguments.md

```diff
@@ -89,10 +89,18 @@ implementation.
 
 ### Parsing KeyBinding Arguments
 
-We'll add a new `IKeyBindingArgs` interface. All current keybinding events will
-be changed from their current types to `TypedEventHandler`s. These
-`TypedEventHandler`s second param will always be an instance of
-`IKeyBindingArgs`. So for example:
+We'll add two new interfaces: `IActionArgs` and `IActionEventArgs`. Classes that
+implement `IActionArgs` will contain all the per-action args, like
+`CopyWhitespace` or `ProfileIndex`. `IActionArgs` by itself will be an empty

```

Comment:
JFYI empty interfaces (AKA marker interfaces) will need to be forced in the idl to have a `uuid`. Otherwise, they don't contain any methods of note and technically don't exist.

Reply:
Yea, I already found that out the hard way. I assumed that wasn't a necessary detail for the spec though (even if 90% of this spec is a technical spec)